source/speech/speech.py

```diff
@@ -969,6 +995,8 @@ def speak(  # noqa: C901
 	import speechViewer
 	if speechViewer.isActive:
 		speechViewer.appendSpeechSequence(speechSequence)
+	if config.conf["braille"]["mode"] == braille.BrailleMode.SPEECH_OUTPUT.value:

```

Comment:
I guess we need an extra extension point here to. See https://github.com/nvaccess/nvda/issues/14520
I think it makes sense to make this a filter. The speech viewer and braille speech mode could just pass through the sequence unfiltered, but an add-on like audio themes might want to apply filters to the sequence.